sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/StateStoreConf.scala

```diff
@@ -97,6 +97,12 @@ class StateStoreConf(
   val enableStateStoreCheckpointIds =
     StatefulOperatorStateInfo.enableStateStoreCheckpointIds(sqlConf)
 
+  /**
+   * Whether the coordinator is reporting state stores trailing behind in snapshot uploads.
+   */
+  val stateStoreCoordinatorReportSnapshotUploadLag: Boolean =
```

Comment:
nit: you can do without this prefix `stateStoreCoordinator` for the val. Same for the RocksDBConf